lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
Actually, it does _not_ appear to mutate even `arguments`:

<img width="511" alt="screen shot 2015-12-02 at 12 59 03 pm" src="https://cloud.githubusercontent.com/assets/445557/11544013/86fb69ea-98f4-11e5-8eb4-71cbb3260730.png">